words_dir/tweet_nouns.py

```diff
@@ -347,7 +347,17 @@
     'flower 🌹',
     'sunflower 🌻',
     'aurora',
-
+    'meadow',
+    'rainforest',
+    'volcano',
+    'petrichor',
+    'cliff',
+    'coast',
+    'cave',
+    'desert,
```

Comment:
@gremlin97 it's okay, it happens to the best of us! We live and we learn, right ? Don't worry about it. Also, I don't mean to be nit-picky but it's just missing a quotation mark now. One more fix 💪

Reply:
:rofl: I will tripple check this time. Thanks for the motivation :v: